blog/2024-06-26.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "📢 Milestone v0.1 has been reached"
```

Comment:
The microphone emoji might be a bit much...

Reply:
I think its fine either way.